README.md

```diff
@@ -130,28 +130,16 @@ Error importing package flask_pdfreactor
 Error importing package flask_eventics
 ```
 
-### 4. Launch the Grunt watch task
+### 4. Launch the Gulp watch task
 
 To watch for changes in the source code and automatically update the running site,
 open a terminal and run:
 
 ```
-$ grunt watch
+$ gulp watch
 ```
 
-*Alternatively, if you don't want to run the full watch task,
-there are two available sub-tasks:*
-
-``` bash
-# Watch & compile CSS only
-$ grunt watch:css
-
-# Watch & compile JS only
-$ grunt watch:js
-
-# Watch & compile CSS & JS only
-$ grunt watch:cssjs
-```
+*There's no longer a need to run each file type on it's own because the watch task only runs for the tasks for files that have changed
```

Comment:
We should remove `There's no longer a need to run each file type on it's own because`, which implies the reader would have read the historical state of the Readme.